### PR TITLE
Minor improvements and bug fixes

### PR DESCRIPTION
**Pull request by Latemus**
Improved parse_mode usage and fixed bug where bob could not display d…
…q stats menu, if leading user in score had markdown element start character in name:

- Problem: parse_mode=MARKDOWN was used by default every time when bob edited an activity host message
- Solution: Moved responsibility of defining parse mode to the caller of 'reply_or_update_host_message' and removed redundant markdown definition from calls (cause of the bug)
- Improvement:
    - command_activity: rename variables in command_activity ('msg' -> 'text' as 'msg' could mean Telegram Message object) and implicated private methods with double underscore
    - Switched all string literal 'Markdown' to constant values from telegram package
    - changed user-message to only have the table wrapped in a code block (same as with dq stats message)
    - dallemini given image caption is now escaped, so special characters won't affect parsing

Improved command triggers and fixed bug where AikaCommand would trigg…
…er when it should not:

- Problem: Different chat_command subclasses had varying regex-matchers for trigger and AikaCommand's trigger did trigger if it was anywhere in the message (for example '.aikamoista' triggered the command)
- Solution: Added requirement, that message should start with '/aika' and only contain it to trigger
- Improvement:
    - Implemented and adopted methods 'regex_simple_command' and 'regex_command_with_parameters' to module command.py. These methods return a unified regex mather so that each ChatCommand subclass does not need to contain details of the regex matcher (exceptions: vai, 1337 and huutista, that have special trigger rules)
    - renamed module 'epic_games' -> 'command_epic_games'. Contains chat_command so might as well use same naming convention than other command modules
    - Created 'assert_command_triggers' method to test_utils.py that makes it easy and fast to test any commands triggering on different messages. Actual handling is replaced with mock which makes this faster. Now all trigger tests are using this new method and old ones have been removed.

Fixed bug where epic games cron job broadcast would fail silently:
- Problem: if timed cron job epic games fetch or data processing failed in any way, bot would fail silently and not broadcast error message
- Solution: refactored implementation so that error message is now sent on epic games cron job broadcast

Fixed bug where dq date confirmation had buttons next to each other r…
…esulting to all text in labels not showing:
- Buttons set to be on top of each other

Improved bot settings menu:
- Now has better help text that states that settings are saved instantly and not after the menu is hid. Bots messages are chatType aware (private/group)
- Now after setting menu is hid, changed settings are listed in the message and the settings menu can be reopened by a button

### CODE REVIEW

File: bobweb/bob/activities/command_activity.py [hunk 1]
```diff
@@ -51,47 +50,53 @@ def change_state(self, state: 'ActivityState'):
         self.state = state
         self.state.execute_state()
 
-    def reply_or_update_host_message(self, msg: str = None, markup: InlineKeyboardMarkup = None):
-        # If first update and no host message is yet saved
-        if self.host_message is None:
-            self.host_message = self.reply(msg, markup)
+    def reply_or_update_host_message(self, text: str = None, markup: InlineKeyboardMarkup = None, parse_mode: str = None):
+        """
+        Important! All user variable values that can contain markdown or html syntax characted should be escaped when
```
Comment (Latemus) [inline bobweb/bob/activities/command_activity.py:55]: characted -> characters

File: bobweb/bob/command_kunta.py [hunk 2]
```diff
@@ -69,7 +68,7 @@ def handle_image_generation_and_reply(update: Update, kunta_name: string, kunta_
         send_image_response(update, kunta_name, image_compilation)
 
     except ImageGenerationException as e:  # If exception was raised, reply its response_text
-        update.effective_message.reply_text(e.response_text, quote=True, parse_mode='Markdown')
+        update.effective_message.reply_text(e.response_text, quote=True, parse_mode=ParseMode.HTML)
```
Comment (Latemus) [inline bobweb/bob/command_kunta.py:73]: parse_mode changed from 'Markdown' -> 'HTML'?

**Review by M4R774 — APPROVED** [review]
Aika iso commit, monta asiaa muuttunut kerralla. Pääpiirteittäin näyttäisi menevän hyvään suuntaan. Mukavasti siirtyy taas boiler platea syvemmälle koodiin. Command_activity.py on ehkä vähän huono nimi, en nimestä pysty arvaamaan mistä on kyse.

**Comment by Latemus** [comment]
@M4R774 Tässä niputettu nämä nyt yhteen ainoastaan, ettei synny turhaa spämmiä commit-viesteistä ryhmiin missä commit-viestien lähetys päällä. Tässä joka commit käytännössä oma kokonaisuutensa, joten voisi mennä hyvin ilman squashaamista dev-haaraan tai sitten voin napata nuo kaikki omiksi haarkoikseen omille merge-requesteille. Tuon 'CommandActivity'n voi uudelleennimetä vaikka 'ChatActivity'ksi tai muuksi ja vastaava nimi moduulille.

**Pull request merged**